working/augmentation-libraries/feature-specification.md

```diff
@@ -710,13 +709,13 @@ top-level scope where declarations are defined. They also share a single private
 namespace. This means that private declarations in the main library or an
 augmentation are visible to all augmentations.
 
-Unlike part files, an augmentation library has its own import scope surrounding
+Unlike part files, an augmentation has its own import scope surrounding that
 that shared top-level scope. Any libraries the augmentation imports are visible
```

Comment:
the augmentation file imports
or
the library augmentation imports

(imports can be treated both syntactically or semantically, either works)